### PR TITLE
Create Github Actions Extension

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@justinpaulson
I created the Github actions extension. Please check and let me know if any issues.
Thank you.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "aha-develop.github-actions",
+  "description": "Github Actions Integration",
```
Comment (trydionel) [inline package.json]:
```suggestion
  "description": "GitHub Actions Integration",
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  "devDependencies": {
+    "@types/react": "^18.0.8",
+    "@types/react-dom": "^18.0.3",
+    "prettier": "^2.6.2"
+  },
+  "ahaExtension": {
+    "contributes": {
+      "views": {
+        "githubActionsAttribute": {
+          "title": "Github Actions",
```
Comment (trydionel) [inline package.json]:
```suggestion
          "title": "GitHub Actions",
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+          "recordTypes": [
+            "Feature",
+            "Requirement",
+            "Epic"
+          ]
+        }
+      },
+      "endpoints": {
+        "githubActionsHook": {
+          "title": "Hook from Github Actions",
```
Comment (trydionel) [inline package.json]:
```suggestion
          "title": "Hook from GitHub Actions",
```

**Review by trydionel — CHANGES_REQUESTED** [review]
Adjusting casing for GitHub

**Review by trydionel — CHANGES_REQUESTED** [review]
This looks great! Could you update the view contribution so clicking on a build opens the details/logs for that build? I'm expecting it to go to a URL like https://github.com/trydionel/babytracker/actions/runs/2246890506, not the project root.

**Comment by ghost** [comment]
@trydionel 
Updated

**Review by trydionel — APPROVED** [review]
Looks good from my side!

File: src/components/AttributeWorkflows.tsx [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    () => workflows.sort((a, b) => new Date(b.finishTime).getTime() - new Date(a.finishTime).getTime()),
+    [workflows]
+  );
+
+  const handleBranchClick = useCallback(
+    (workflow: IWorkflow) => {
+      if (!isValidExternalLink(workflow?.url)) {
+        return;
+      }
+      window.open(workflow?.url, '_blank', 'noopener,noreferrer');
```
Comment (cornu-ammonis) [inline src/components/AttributeWorkflows.tsx:25]: 👍

**Review by cornu-ammonis — COMMENTED** [review]
Looks good from a security perspective!

**Review by justinpaulson — APPROVED** [review]

**Pull request merged**